bodhi/server/schemas.py

```diff
@@ -807,3 +807,8 @@ class WaiveTestResultsSchema(CSRFProtectedSchema, colander.MappingSchema):
         colander.String(),
         missing=None,
     )
+
+    waive_tests = colander.SchemaNode(
+        colander.String(),
+        missing=None,
+    )
```

Comment:
I recommend just naming this ```tests``` as well, since the API already implies that waiving is happening.

Also, I don't think the tests will come in as a list of strings with this schema, but just as a string.

Reply:
Name adjusted.

> Also, I don't think the tests will come in as a list of strings with this schema, but just as a string.

I was a little afraid of this, but I didn't find how to ask for a list, any clues for me?